node-sso-example/routes/index.js

```diff
@@ -16,7 +16,7 @@ app.use(
 
 const workos = new WorkOS(process.env.WORKOS_API_KEY)
 const clientID = process.env.WORKOS_CLIENT_ID
-const connection = ''
+const connection = 'conn_01G2TM1BYXCPFB5Y12WN7FK2DY'
```

Comment:
These updates are looking great! The only thing sticking out to me is that a real connection ID appears to be hard-coded in here. Is that intentional, or should we leave it as a blank string?

Reply:
Thanks for catching it! I'll remove it now, only added when testing :)